FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -283,4 +321,23 @@ NS_SWIFT_NAME(RemoteConfig)
 ///                         nil if the key doesn't exist in the default config.
 - (nullable FIRRemoteConfigValue *)defaultValueForKey:(nullable NSString *)key;
 
+#pragma mark - Realtime
+
+/// Completion handler invoked by fetch methods when they get a response from the server.
+///
+/// @param configUpdate Information on which key's values have changed
+/// @param error  Error message on failure.
+typedef void (^FIRRemoteConfigUpdateCompletion)(FIRRemoteConfigUpdate *_Nullable configUpdate,
+                                                NSError *_Nullable error)
+    NS_SWIFT_UNAVAILABLE("Use Swift's closure syntax instead.");
```

Comment:
Not sure why this is marked Swift unavailable? 
What's the alternative Swift way of calling this?

Reply:
Completion typedef's should not be exposed to Swift, since they're less readable than spelling out the original types.